packages/component-library/components/Button/styles.scss

```diff
@@ -440,4 +441,11 @@ $caButton-verticalPaddingForm: calc(
   &:active span {
     color: $kz-color-coral-600;
   }
+
+  &:hover,
+  &:active,
+  &:global(.focus-visible) {
+    color: $kz-color-coral-600 !important;
+    background: $kz-color-coral-100 !important;
```

Comment:
Curious if there's something else you can do here rather that `important`. Could you increase the specicifity instead?

Reply:
Ah, I had forgotten about this. I think I did it because I couldn't find a way to increase the specificity. I'll have a look now